### PR TITLE
adding hometown file for jrdisc2

### PR DESCRIPTION
**Pull request by jrdisc2**
@Mogle1 what kind of restaurants do you like so I can add them to the hometown file?

### CODE REVIEW

File: jrdisc2-Brazil.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Rio de Janeiro, Brazil
+
+### Restaurants
+- Local All you can eat Steak Houses
+- Juice Bars
+- Galeterias (small grilled chics specialty restaurant)
+
```
Comment (Mogle1) [inline jrdisc2-Brazil.md]: I really like Steaks and :pizza:

**Review by Mogle1 — CHANGES_REQUESTED** [review]
Add some restaurants that I would like to go to and this will be ready to :ship:

**Comment by jrdisc2** [comment]
Even though I dont know any specific Pizza place in Rio, there are tons of them in there.

**Review by Mogle1 — APPROVED** [review]
This looks good. good to :ship:

**Review by jrdisc2 — COMMENTED** [review]
Done!

**Pull request merged**